content/tracing/services/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Services list
+kind: Documentation
+---
+
+## Overview
+
+After having [instrumented your application](/tracing/languages), your reporting services are listed on [the APM services page](https://app.datadoghq.com/apm/services).  
+Select a service to have more performance insights, [read our dedicated service documentation to learn more](tracing/services/service)
+
+{{< img src="tracing/services/services_page.png" alt="Services page" responsive="true" popup="true">}}
+
+## Filtering the service list
+
+The services list is a bird eyed view of all [services](/tracing/services/service) reporting from your infrastructure. It can be filtered depending on:
+
+* [Your environment](#environment).
+* [Your service type](#services-types)
+* A query
+
+{{< img src="tracing/services/services_filtering.gif" alt="Services filtering" responsive="true" popup="true" style="width:75%;">}}
```

Comment:
In this image can we remove the cursor?  Also can we maybe highlight the areas where you can filter (env dropdown, search bar, type buttons)?

Reply:
This is actually a gif, but git are not running currently on staging --' a fix is on its way